### PR TITLE
feat: optimize deploy addon error message alerts

### PR DESCRIPTION
**Pull request by CeerDecy**
#### What this PR does / why we need it:

optimize deploy addon error message alerts

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=570070&iterationID=12783&type=TASK)

#### Specified Reviewers:

/assign @iutx 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |       optimize deploy addon error message alerts       |
| 🇨🇳 中文    |       Addon 部署预检查优化版本提示       |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by iutx — APPROVED** [review]
LGTM. @sfwn

**Comment by CeerDecy** [comment]
> @CeerDecy Please ensure the CI process.

done

File: internal/tools/orchestrator/services/addon/addon_handler.go [hunk 1]
```diff
@@ -127,7 +127,12 @@ func (a *Addon) GetAddonExtention(params *apistructs.AddonHandlerCreateItem) (*a
 		if emptyVersion {
 			err = errors.New(i18n.OrgSprintf(params.OrgID, AddonDefaultVersionDoseNoExist, params.AddonName))
 		} else {
-			err = errors.New(i18n.OrgSprintf(params.OrgID, AddonVersionDoseNoExist, params.AddonName, version))
+			def, ok := addons.GetDefault()
```
Comment (sfwn) [inline internal/tools/orchestrator/services/addon/addon_handler.go:130]: Clearly the main purpose of this code snippet is to obtain the recommendVersion:

```golang
recommendVersion := "-"
if def, ok := addons.GetDefault(); ok {
  recommendVersion = def.Version
}
```

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**